acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/KeyLoadAndSignAcceptanceTest.java

```diff
@@ -128,6 +133,34 @@ public void unusedFieldsInRequestDoesNotAffectSigning() throws Exception {
     assertThat(response.getStatusCode()).isEqualTo(HttpResponseStatus.OK.code());
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {"/signer/block"}) // , "/signer/attestation"})
```

Comment:
We want to test this for all endpoints i.e. block, attestation, randao_reveal not just block

Reply:
done